client/src/pages/trainer/learning-paths/create/index.tsx

```diff
@@ -47,7 +54,40 @@ const LearningPathCreate = (): JSX.Element => {
         }
         return await trigger(['name', 'description', 'category']);
       case 2:
-        return await push('/trainer/learning-paths');
+        try {
+          const formData = {
+            ...values,
+            category: values.category.map(({ id }) => id),
+            courses: values.courses.map(({ id, order }) => {
+              return {
+                course: id,
+                course_order: order,
+              };
+            }),
+            is_active: values.isActive,
+            image:
+              typeof values.image === 'string'
+                ? '/' + values.image
+                : values.image?.name
+                ? '/' + values.image.name
+                : null,
+          };
+
+          const res: any = await createLearningPath(formData);
+          if ('error' in res) {
+            const { data } = res.error;
+            const property = Object.keys(data)[0];
+            throw new Error(data[property][0]);
+          } else {
+            alertSuccess('Learning path created successfully!');
+            await push('/trainer/learning-paths');
+            dispatch(reset());
+          }
+        } catch (e: any) {
+          alertError(e.message);
```

Comment:
Please give it a custom error message like `something went wrong`, since you are passing the error from BE, it may not be understandable by users, I see its returning something like `[object object]` or just `< ` as error message right now